staff/lab/lab-wakeup

```diff
@@ -21,7 +23,18 @@ def wake_hosts(filter, quiet):
         ['wakeonlan'] + [host['macAddress'][0] for host in hosts_by_filter(filter)],
         stdout=subprocess.DEVNULL if quiet else sys.stdout,
     )
+    
+def is_atdp_hours(when=None):
+        """Return whether the given time is during atdp hours.
 
+        If not provided, when defaults to now.
+        """
+        days = [0, 2, 4] #Monday, Wednesday, Friday
```

Comment:
Nit: suggested comment style: two spaces between the code and the comment, space after the comment character